core/zms/src/main/java/com/yahoo/athenz/zms/Assertion.java

```diff
@@ -20,6 +20,9 @@
     @RdlOptional
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
     public Long id;
+    @RdlOptional
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public Boolean caseSensitive;
```

Comment:
I didn't want to change the API. Users that would like to save assertion as case-sensitive will need to set the assertion as caseSensitive as part of the structure passed to Athenz in putAssertion and putPolicy.
Please note the following:
1. This value will not be saved to DB. It will only be used to tell ZMS to keep the resource and action as case sensitive
2. The policy structure will also have this field. If it is set, all assertions will be lowercased